packages/vscode-extension/src/debugging/RadonCDPProxyDelegate.ts

```diff
@@ -250,9 +250,9 @@ export class RadonCDPProxyDelegate implements CDPProxyDelegate {
 
     try {
       const sourceMapData = await this.getSourceMapData(sourceMapURL);
-      const isMainBundle = sourceMapData.sources.some((source: string) =>
-        source.includes("__prelude__")
-      );
+      const isMainBundle =
+        sourceMapData.sources &&
+        sourceMapData.sources.some((source: string) => source.includes("__prelude__"));
```

Comment:
```suggestion
      const isMainBundle =
        sourceMapData.sources !== undefined ?
        sourceMapData.sources.some((source: string) => source.includes("__prelude__")) : false;
```
or

```suggestion
      const isMainBundle =
        sourceMapData.sources?.some((source: string) => source.includes("__prelude__")) ?? false;
```
or even

```suggestion
      if (sourceMapData.sources === undefined) {return command;}
      const isMainBundle =
        sourceMapData.sources.some((source: string) => source.includes("__prelude__"));
```
since we wouldn't fire the event before, and maybe we don't want to in this case.